LongestPalindrome.java

```diff
@@ -0,0 +1,36 @@
+//Time Complexity: O(n)
+//Space Complexity: O(n)
```

Comment:
space used by map will be contsnat as only lower case characters are there

Reply:
I have updated the space complexity here, thank you